pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/json/users/models/EmailAddress.java

```diff
@@ -0,0 +1,21 @@
+package org.broadinstitute.ddp.json.users.models;
+
+import javax.validation.constraints.Email;
+
+import lombok.Data;
+import org.apache.commons.validator.routines.EmailValidator;
+
+@Data
+public class EmailAddress {
+
+    @Email
+    private final String value;
+
+    public EmailAddress(String email) {
+        if (EmailValidator.getInstance().isValid(email) == false) {
+            throw new IllegalArgumentException("email format is invalid");
+        }
```

Comment:
Quick note on the validation here- The Apache and Hibernate validators accept different formats for the email address.

The `javax.validation.constraints.Email` validator allows for `local-part@the-rest` (a `@` is required), while the Apache EmailValidator requires something in the form of `local-part@domain.part` (A `@` and `.` is required). I don't think we're likely to run into issues with this in normal use, but it is a distinction to be aware of.

Suggestions on how to invoke the javax validator outside of this class be serialized/deserialized (currently, the checking is only done on the request payload by the `ValidatedJsonInputRoute` class)